office-js/office-js-tests.ts

```diff
@@ -0,0 +1,175 @@
+/*
+------------------------------------------ START OF LICENSE -----------------------------------------
```

Comment:
https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE 
this repository is a MIT lisence